ScoutSuite/cli_parser.py

```diff
@@ -197,13 +197,19 @@ def __init__(self, default_args=None):
 
         self.parser.add_argument('--project-id',
                                  action='store',
+                                 required='--all' not in sys.argv,
```

Comment:
I believe this is the "old" parser, which is being refactored in https://github.com/nccgroup/ScoutSuite/pull/151